src/shared/components/column-select/__tests__/__snapshots__/ColumnSelect.spec.tsx.snap

```diff
@@ -7,14 +7,14 @@ exports[`ColumnSelect component should render 1`] = `
   >
     <div
       class="column-select-drag-drop__list"
-      data-rfd-droppable-context-id=":r0:"
+      data-rfd-droppable-context-id="1"
```

Comment:
The library was already using React's useId, but here the snapshot changed because it is now mocked in tests just like for the usage of it in our own codebase